Samples/Basic/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
```

Comment:
Why does Samples/Basic get it's own .eslintrc.js configuration?

Reply:
Because I need to turn of the no-restricted-globals rule, but didn't want to to that everywhere. Was a bit quick here, so I'll change it to only do that

Reply:
You can write an override in the root config that only globs Samples for example 😁

Reply:
That's also an option, but I kind of like having the config close to what it overrides. Made it a bit simpler now.